bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/device/MessageHandler.java

```diff
@@ -971,16 +971,20 @@ public int transform(int raw) {
             return (raw);
         }
 
-        private int extractValue(Msg msg) throws FieldException {
+        private int extractValue(Msg msg, int group) throws FieldException {
             String lowByte = getStringParameter("low_byte", "");
-            if (lowByte == "") {
+            if (!lowByte.equals("")) {
```

Comment:
Here are the jar and .deb files with the fix. I hope the links work.

https://drive.google.com/file/d/0B8OtzwATWQSWbzJzUGNCVldmczA/view?usp=sharing
https://drive.google.com/file/d/0B8OtzwATWQSWcDNWenpOYVRKWms/view?usp=sharing